scripts/Units/DemoUnit.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UnityEngine;
+using Unity.VisualScripting;
+using DolbyIO.Comms;
+
+namespace DolbyIO.Comms.Unity
+{
+    [UnitTitle("Demo Conference")]
+    [UnitCategory("DolbyIO")]
+    public class DemoUnit : ConferenceUnit
+    {
+        [DoNotSerialize]
+        [PortLabelHidden]
+        public ControlInput inputTrigger;
+
+        [DoNotSerialize]
+        [PortLabelHidden]
+        public ControlOutput outputTrigger;
+
+        [DoNotSerialize]
+        public ValueInput SpatialAudio;
+
+        protected override void Definition()
+        {
+            base.Definition();
+
+            inputTrigger = ControlInputCoroutine("inputTrigger", Demo);
```

Comment:
I like to use the notation `nameof(inputTrigger)` for this kind of code.

Reply:
Right, this is a copy/paste error. Anyway those field names are hidden. Let me fix